### ISSUE TITLE
have a problem when finetune train_tripletloss.py with pretrained_model

### ISSUE THREAD

**Issue by clscy** [davidsandberg/facenet#759]
Hi,
I try to train with pretrained_model(tendorflow1.4.1, model:[20180402-114759](https://drive.google.com/open?id=1EXPBSXwTaqrSC0OhUdXNmKSh9qJUQ55-)),
python train_tripletloss.py --logs_base_dir ~/logs/facenet/ --models_base_dir ~/models/facenet/ --data_dir ~/data/vggface2/train_160 --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir ~/data/lfw --lfw_pairs ~/data/pairs.txt --optimizer RMSPROP --learning_rate 0.01 --weight_decay 1e-4 --max_nrof_epochs 500 --people_per_batch 6 --pretrained_model /home/Github36/facenet/src/20180402-114759/model-20180402-114759.ckpt-275

in ~/data/vggface2/train_160, only have 12 folder(only a little dataset). 
then have a problem

Restoring pretrained model: /home/chengls/Github36/facenet/src/20180402-114759/model-20180402-114759.ckpt-275
2018-05-25 11:36:32.468627: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Assign requires shapes of both tensors to match. lhs shape= [128] rhs shape= [512]
	 [[Node: save/Assign_20 = Assign[T=DT_FLOAT, _class=["loc:@InceptionResnetV1/Bottleneck/BatchNorm/moving_variance"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](InceptionResnetV1/Bottleneck/BatchNorm/moving_variance, save/RestoreV2_20/_1549)]]
2018-05-25 11:36:32.469296: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Assign requires shapes of both tensors to match. lhs shape= [128] rhs shape= [512]

I think in train_tripletloss.py the "saver.restore(sess, os.path.expanduser(args.pretrained_model))" cannot restore the model-20180402-114759.ckpt-275.
How can I solve this problem?
thanks.

**Comment by aawanghuiaa** [comment]
add argument --embedding_size 512

**Comment by clscy** [comment]
OK,thank you @aawanghuiaa .
